apps/omg_watcher/lib/omg_watcher/api/utxo.ex

```diff
@@ -42,4 +40,19 @@ defmodule OMG.Watcher.API.Utxo do
   def create_challenge(utxo) do
     ExitProcessor.create_challenge(utxo)
   end
+
+  @decorate measure_event()
+  @spec compose_utxo_exit(Utxo.Position.t()) :: {:ok, exit_t()} | {:error, :utxo_not_found}
+  def compose_utxo_exit(decoded_utxo_pos) when is_deposit(decoded_utxo_pos) do
```

Comment:
As we know `API` works on domain structures - I'd drop `decoded` as obvious.
```suggestion
  def compose_utxo_exit(utxo_pos) when is_deposit(utxo_pos) do
```